cmd/kubectl-access/kubectl-access.go

```diff
@@ -323,6 +323,28 @@ func (ac *accessCommand) Grant(cmd *cobra.Command, requestName string) error {
 		return fmt.Errorf("could not print object: %w", err)
 	}
 
+	duration := "once"
+	if accessRequest.Spec.ValidFor != "" {
+		duration = fmt.Sprintf("for %s", accessRequest.Spec.ValidFor)
+	}
+	command := fmt.Sprintf("%q", accessRequest.Spec.ExecOptions.Command)
+	if len(accessRequest.Spec.ExecOptions.Command) == 0 {
+		command = "ANY command"
+	}
+	fmt.Println("---")
+	fmt.Printf(`%s requested by %q
+
+- requesting access to %q and container %q in namespace %s
```

Comment:
I would also quote the namespace here

Reply:
Yeah. I just thought about quoting it for consistency. Since the other 2 fields are quoted, it might make sense to do it for all

Reply:
Okay, quoted the fields now.